bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/SatelModule.java

```diff
@@ -88,7 +88,11 @@
      */
     protected static class ConnectionFailureException extends Exception {
 
-        private static final long serialVersionUID = 1L;
+        private static final long serialVersionUID = 2L;
```

Comment:
That is indeed how its supposed to be used (but most people tent to forget to change it )